src/bikeracks/schema.sql

```diff
@@ -80,9 +80,10 @@ CREATE TABLE suggested_removals (
 /* Maps reason integer from suggested_removals table to a language
 and reason */
 CREATE TABLE removal_reasons (
-    reason_id INTEGER PRIMARY KEY AUTOINCREMENT,
+    removal_reason_id INTEGER PRIMARY KEY AUTOINCREMENT,
     lang TEXT DEFAULT "en",
-    reason TEXT
+    reason TEXT,
+    FOREIGN KEY(removal_reason_id) REFERENCES suggested_removals(reason)
```

Comment:
The foreign key should have a matching name.

Call them both `removal_reason_id` or `reason_id` or whatever it is you prefer.

Reply:
Done.